spec/support/blueberry_rails_helpers.rb

```diff
@@ -0,0 +1,42 @@
+require 'fileutils'
+
+module BlueberryRailsHelpers
+  APP_NAME = 'test_project'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.